docs/edge_processor.md

```diff
@@ -33,12 +33,12 @@ stateDiagram
     EP --> Dest
 ```
 
-## Basic Setup
+## Set up the Edge Processor for SC4S
 
-### Docker / Podman
+### Set up on Docker / Podman
 
-Setup on your `env_file` HEC URL as IP of EP worker EC2 instance.
-Token you can find in EP "global settings" page.
+1. In the `env_file`, configure the URL address to the HEC receiver of the selected Edge Processor instance. The HEC URL of the EP instance is the same as the IP of the hosting machine.
```

Comment:
@ikheifets-splunk can you suggest a final version for this part? I think that EP worker doesn't have to be installed on EC2 instance, it can be any other cloud's compute unit or someone's local host

Reply:
But a worker doesn't have to be installed on EC2 right? It can be any type of machine, local host or GCP's GCE

Reply:
@mstopa-splunk it might be anything, but it's most clear explanation what does it mean, if you know better explanation please propose. In UI they call it `Edge Processor`, it's very common name

Reply:
I've updated the text, just let me know if you have any more questions